models/models.js

```diff
@@ -6,8 +6,9 @@ exports.selectTopics = () => {
     })
 };
 
-exports.selectArticles = () => {
-    return db.query(`SELECT * FROM articles;`).then(articles => {
+exports.selectArticles = (sort_by = "created_at", order = "desc") => {
+    return db.query(`SELECT * FROM articles
+                    ORDER BY ${sort_by} ${order};`).then(articles => {
```

Comment:
This is fine for the time being - although over engineered in this circumstance. Later when these queries can be provided by a user should this model take arguments